lib/fluent/plugin/storage_local.rb

```diff
@@ -64,7 +64,8 @@ def configure(conf)
             end
           end
         elsif root_dir = owner.plugin_root_dir
-          @path = File.join(root_dir, 'storage.json')
+          basename = (conf.arg && !conf.arg.empty?) ? "storage.#{conf.arg}.json" : "storage.json"
```

Comment:
`conf.arg` can contain space like `<storage foo bar>`.
So need to validate or convert unexpected charactors.